app/src/components/Hand.vue

```diff
@@ -23,30 +35,49 @@
 <script>
 import { CardType } from "../bf-game";
 import Card from "./Card";
+import Button from "./Button";
 
 export default {
   name: "Hand",
   components: {
-    Card
+    Card,
+    Button
   },
   data() {
     return {
       CardType
     };
   },
+  computed: {
+    lastIndex() {
+      return this.deckCount - 1;
+    },
+    randomIndex() {
```

Comment:
In some cases, the `randomIndex` computed property wont be "re-rolled" : computed properties functions are assumed to be pure and their result is cached and recalculated only when their dependencies change.

Consider generating random numbers when you need them instead of "in advance". Here, I would make a `insertPerishAtRandom` method.

Reply:
Done :+1: